core/drivers/bcm_hwrng.c

```diff
@@ -44,29 +48,74 @@ static void bcm_hwrng_reset(void)
 			RNG_CTRL_MASK, RNG_CTRL_ENABLE);
 }
 
-uint32_t bcm_hwrng_read_rng(uint32_t *p_out, uint32_t words_to_read)
+static uint32_t do_rng_read(void *buf, ssize_t blen)
```

Comment:
Why are the `uint32_t` and `ssize_t` types used here? I'd normally expect `size_t`.